### PR TITLE
Add sponsor blog for EPAM

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @vipulgupta2048 @sayanchowdhury @palnabarun

### CODE REVIEW

**Comment by inovizz** [comment]
LGTM.

File: content/blog/2020-announcing-epam-as-gold-sponsor.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Category: 2020
+Slug: sponsor-announcement-epam
+Summary: We are happy to introduce **EPAM** as a Gold Sponsor for PyCon India 2020. 
+
+We are happy to introduce **EPAM** as a Gold Sponsor for PyCon India 2020.
+
+<br>
+<div class="text-center">
+  <a href="https://welcome.epam.in/" target="_blank">
+    <img src="https://user-images.githubusercontent.com/3849885/93663821-126f4600-fa88-11ea-982a-2d3feaba4843.png" alt="EPAM">
```
Comment (palnabarun) [inline content/blog/2020-announcing-epam-as-gold-sponsor.md:13]: This change. Please use https://in.pycon.org/2020/assets/images/sponsors/epam.png

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: content/blog/2020-announcing-epam-as-gold-sponsor.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-announcing-epam-as-gold-sponsor.md]: Done

**Comment by Sc0rpi0n101** [comment]
Ping @palnabarun

**Review by palnabarun — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**